### PR TITLE
Added the module.exports declaration to the angular-dfp script.

### PR DESCRIPTION
**Pull request by jasonconway-williams**

### CODE REVIEW

File: angular-dfp.js [hunk 1]
```diff
@@ -439,3 +439,4 @@ angular.module('ngDfp', [])
     };
   }]);
  
+module.exports = 'ngDfp';
```
Comment (ianmurrays) [inline angular-dfp.js]: Won't this throw an error in browsers if using bower?

Comment (jasonconway-williams) [inline angular-dfp.js]: Sorry Ian, your are correct. I have now removed this change and implemented a better way of doing it, utilising the npm way of handling modules.

I have added an index.js file that requires the angular-dfp.js script and exports the module name. The index file is referenced in the package.json file as the main file. This allows any npm/common.js users to implement the angular-dfp module as an npm/commonjs module but it also allows non npm/commonjs users to use the file in the standard way by referencing the angular-dfp scipt in html.

**Comment by ianmurrays** [comment]
Looks good. Thanks!

**Comment by jasonconway-williams** [comment]
Ian at the moment, the module can be installed via bower or as a github url via npm. Is there any change you can publish the module to the npm repository? This should just be a simple case of:

* From a terminal, create a user with the **npm adduser** command and login via the **npm login** command.
* Run the **npm publish** command to publish the module. Once published the module should be available at **https://npmjs.com/package/ngDfp**.

This would allow users to install ngDfp via npm with the command npm install ngDfp

**Comment by ianmurrays** [comment]
Hey @jasonconway-williams it's done 😉

**Comment by jasonconway-williams** [comment]
Cheers @ianmurrays

**Pull request merged**